Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Multi-stage build for Claude Code UI
+
+# Stage 1: Build frontend
+FROM node:20-alpine AS frontend-builder
+
+WORKDIR /app
+
+# Copy package files
+COPY package*.json ./
+
+# Install dependencies (including devDependencies for build)
+RUN npm ci
+
+# Copy frontend source
+COPY index.html ./
+COPY vite.config.js ./
+COPY postcss.config.js ./
+COPY tailwind.config.js ./
+COPY public/ ./public/
+COPY src/ ./src/
+
+# Build frontend
+RUN npm run build
+
+# Stage 2: Setup backend and runtime
+FROM node:20-alpine AS runtime
+
+# Install system dependencies
+RUN apk add --no-cache \
+    curl \
+    git \
+    openssh-client \
+    python3 \
+    make \
+    g++ \
+    sqlite
+
+# Create app user
+RUN addgroup -g 1001 -S nodejs && \
+    adduser -S nodejs -u 1001
+
+WORKDIR /app
+
+# Copy package files
+COPY package*.json ./
+
+# Install production dependencies only
+RUN npm ci --only=production && \
```

Comment:
`--only=production` is deprecated? Should be `--omit=dev`? See https://docs.npmjs.com/cli/v11/commands/npm-ci#omit